### DISCUSSION TITLE
reportDate timezone

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#31] [Q&A]
We are passing the report date for this API `/v2/sp/{recordType}/report`. In docs, it's mentioned as the timezone is taken from the profile.  So do we receive the ads data in the profile timezone? 

for instance, the profile timezone is `America/Los_Angeles` and the report request date is 20221002. So the API returns the data for this date in `America/Los_Angeles` timezone?

Can anyone help with this?

**Comment by elizafarley** [comment]
Hi @Nilanth, you have the right understanding on this. The report contains data based on the date requested, in the timezone attached to the profile.
